### PR TITLE
Chavanr/e2e delete workspace

### PR DESCRIPTION
**Pull request by chavanr2019**
Description:
Created a test case to delete workspace using the Ellipsis menu action on the card :
- delete an existing workspace or 
- create a new workspace one if none exists

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test:local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: e2e/tests/workspace/workspace-delete.spec.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+   * Test:
+   * - Find an existing workspace. Create a new workspace if none exists.
+   * - Select "Delete" thru the Ellipsis menu located on the Workspace card.
+   * - Get the content on the modal
+   * - Click the delete workspace button on the modal
+   * - Navigate to the "View Workspaces" page
+   * - Verify that the workspace has been delete
+   */
+  
+  describe('From "Your Workspaces" page using Workspace card ellipsis menu', () => {
```
Comment (aweng98) [inline e2e/tests/workspace/workspace-delete.spec.ts:24]: I think it's the `Home` page where delete happens, right?

File: e2e/tests/workspace/workspace-delete.spec.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+      // Verify Workspace Access Level is OWNER.
+      const accessLevel = await workspaceCard.getWorkspaceAccessLevel();
+      expect(accessLevel).toBe(WorkspaceAccessLevel.Owner);
+      
+      await workspaceCard.asElementHandle().hover();
+      // click on Ellipsis "Delete"
+      await (workspaceCard.getEllipsis()).clickAction(EllipsisMenuAction.Delete, { waitForNav: false });
+
+      const modal = new Modal(page);
```
Comment (aweng98) [inline e2e/tests/workspace/workspace-delete.spec.ts:40]: Can you create a reusable function for delete workspace modal similar to this one? It's okay if you don't feel comfortable doing it as this is your first PR.
```
/**
   * Click DELETE COHORT button in Cohort Delete Confirmation dialog.
   * @return {string} Dialog textContent.
   */
  async deleteConfirmationDialog(): Promise<string> {
    const modal = new Modal(this.page);
    const contentText = await modal.getContent();
    await modal.clickButton(LinkText.DeleteCohort, {waitForClose: true});
    await waitWhileLoading(this.page);
    return contentText;
  }
```

File: e2e/tests/workspace/workspace-delete.spec.ts (hunk 1 above)
Comment (chavanr2019) [reply e2e/tests/workspace/workspace-delete.spec.ts:24]: the delete happens on"YOUR WORKSPACES" page.

File: e2e/tests/workspace/workspace-delete.spec.ts (hunk 2 above)
Comment (chavanr2019) [reply e2e/tests/workspace/workspace-delete.spec.ts:40]: working on it!

Comment (chavanr2019) [reply e2e/tests/workspace/workspace-delete.spec.ts]: Done!

**Comment by calbach** [comment]
Hi Rohini, congrats on the first PR!

Some useful links/notes going forward:
- Please be sure to use a meaningful PR title going forward, this helps when looking through the history etc to identify what a change was doing, without reading the whole PR or description. In this case, probably something like `[risk=no] Add puppeteer test for workspace deletion`
- Please include a `[risk=]` tag in the title, per https://github.com/all-of-us/workbench/blob/acdc387a7a21c99ab09747c2f11ef22929cba220/.github/CONTRIBUTING.md ; for any PR that simply adds or changes puppeteer tests, you can just use `[risk=no]`
- Please include `[RW-...]` in the title if there is an associated ticket
- PR flow guidelines: https://docs.google.com/document/d/1fnQ9FAy1_S9YBRTA0NGpY7ZhJ9I-fhr13iMZO25BWFM/edit

The last link there also includes some more details about some of the usual git flows that you may need to use. Feel free to reach out if you have any questions on this stuff.

**Pull request merged**